### ISSUE TITLE
[FEAT] 인식 모델 연결 — 구간 판정, 임계값, 기동 시 프리로드

### ISSUE THREAD

**Issue by Yewon-Ko1** [lucky7terry/GoldenSign#47]
## 📌 개요

#42 로 단어 구간을 모아 모델 입력 `(60, 420)` 을 만드는 것까지 됐다.
이제 그 배열을 실제로 모델에 넣어 단어를 돌려준다.

`word_end` 의 `text` 가 아직 항상 `null` 이다. 서버는 좌표만 뽑고 있고,
`/health` 의 `loaded` 도 하드코딩된 `false` 다. 이걸 실제 상태로 바꾼다.

판정만 붙이면 되는 것이 아니라 **언제 단어를 주장하지 않을지**를 정해야 한다.
softmax 는 합이 1 이라 모르는 동작을 넣어도 어딘가에 확률이 몰린다.

## 🛠️ 작업 내용

**추론 연결**
- [ ] `recognition_service` — `(60, 420)` 을 받아 50개 클래스 중 하나를 고른다
- [ ] `word_end` 결과에 실제 `text` 를 싣는다
- [ ] `make_predictor` 결과를 캐시한다. 지금은 호출마다 새 `tf.function` 과
      그래프를 만들어서, 단어마다 부르면 그래프가 쌓이고 트레이싱 비용을 다시 낸다

**임계값**
- [ ] 확신도와 **2위와의 격차**를 함께 본다. 확신도만 보면 50개가 고르게
      나뉜 경우와 두 개가 팽팽한 경우를 구분하지 못한다
- [ ] 못 넘으면 `text: null`. 자리표시자 문자열을 넣으면 그게 그대로
      안경 화면에 뜬다
- [ ] 거절당했을 때 무엇이 1위였는지 결과에 남긴다. 없으면 임계값을
      조정할 때 왜 안 나왔는지 알 수 없다

**상태 보고**
- [ ] `get_model_health_status` 가 실제 로딩 상태를 읽는다. 좌표 추출과
      인식 모델은 따로 실패할 수 있으므로 상태가 셋이다:
      `unavailable` / `keypoints_only` / `recognition`
- [ ] 실패 사유를 `/health` 에 그대로 싣지 않는다. 서버 절대경로가 들어 있고
      `/health` 는 인증이 없다

**기동**
- [ ] `lifespan` 에서 모델을 미리 올린다. 3~8초가 걸려서 첫 단어에서 올리면
      그 사용자만 그 시간을 통째로 기다린다
- [ ] 모델 파일이 없어도 기동은 계속한다. 좌표 추출은 그대로 동작해야 한다

**검증 · 문서**
- [ ] 서버 코드를 그대로 부르는 검증 스크립트. 기존 `verify_from_video.py` 는
      파이프라인을 재구성해서 확인하는 도구라 서버 경로 자체를 증명하지 못한다
- [ ] 임계값 테스트 — 통과해야 할 값과 거절되어야 할 값을 코드에 고정
- [ ] `README` 에 모델 파일을 어디에 두는지. 저장소에 없는 파일이라
      clone 한 사람이 알 방법이 없다
- [ ] `docs/api-contract.md` 의 `dev-0.4` 결과 예시 갱신

## 🔗 참고 자료 / 기타

- #42 (단어 구간 입력 파이프라인) 이후에 진행한다. `(60, 420)` 이 있어야 붙일 수 있다
- 임계값 근거는 `word-mode-decision.md` 의 검증이다. 단어 단위 결과가
  최저 확신도 0.615, 2위와의 격차 최저 0.377 이라 5시점 전부 통과한다.
  반대로 슬라이딩 윈도우의 애매한 판정(0.366, 격차 0.083)은 거절되어야 한다

```
  RECOGNITION_CONFIDENCE_THRESHOLD = 0.5
  RECOGNITION_MARGIN_THRESHOLD = 0.15
```

- `.keras` 파일은 `.gitignore` 에 있어 저장소에 없다. 배포 방법(GitHub
  Releases / LFS / 수동)은 이 이슈 범위 밖이고 배포 준비에서 정한다.
  여기서는 `server/models/` 에 두면 된다는 것만 문서화한다
- 모델이 없을 때 `word_end` 는 `text: null` 을 돌려주고 `model.loaded` 가
  `false` 다. 앱은 이미 그 상태를 다루고 있다
